src/analysis/Analyses.ts

```diff
@@ -520,12 +514,12 @@ export const BaseAnalyses = (
         .Workspaces.workspace(workspaceInfo.namespace, workspaceInfo.name)
         .listActiveFileTransfers();
       setActiveFileTransfers(!_.isEmpty(fileTransfers));
-
-      await refreshRuntimes();
     });
-
-    load();
-  });
+    if (workspace?.workspaceInitialized) {
```

Comment:
waiting until the workspace is initialized means we aren't loading based off of incorrect or incomplete state

Reply:
Hmm... it's possible we may be able to more robustly accomplish this using the `LoadedState` pattern we have elsewhere. Not necessarily adamant about this idea, and haven't fully thought through if its applicable since this is on a specific 'selected' workspace as opposed to the overall workspace store, but it has some nice benefits down the line (i.e. condensing the logic to display loading spinners) 

Definition (assuming this would live on the workspace object/store): https://github.com/DataBiosphere/terra-ui/blob/dev/src/analysis/useAnalysisFiles.ts#L44
Initialization: https://github.com/DataBiosphere/terra-ui/blob/dev/src/analysis/Analyses.ts#L473 (follow
Usage: https://github.com/DataBiosphere/terra-ui/blob/dev/src/analysis/Analyses.ts#L537

Reply:
`workspaceInitialized` is more complex than just "have we gotten a response back from the server". There are multiple calls that we are dealing with (bucket location, storage information). Perhaps I didn't name the property very well.

I did try using `LoadedState` when I first introduced `workspaceInitialized` and couldn't figure out how to make it work. I'm not against someone else giving it a try though.

Reply:
More details: Part of what I was dealing with is the historical pattern that default values are used for GCP bucket information before/even if we cannot obtain the actual values. I needed to differentiate between what are "real" values and what are simply defaults. And sometimes we cannot get those values for quite a while due to GCP permissions syncing or Azure cloning being async. And we also have to handle the expected failure case of requesterPays.